src/redux/books/booksSlice.js

```diff
@@ -1,11 +1,53 @@
-import { createSlice } from '@reduxjs/toolkit';
-import booksList from '../../booksList';
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+// import booksList from '../../booksList';
+
+const app_id = 'FUmEw9612WnSTelBC867';
+const url = `https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/${app_id}/books`;
 
 const initialState = {
-  book: booksList,
+  book: [],
   isLoading: true,
+  error: null,
 };
 
+export const getBooks = createAsyncThunk(
+  'books/getBooks',
```

Comment:
- [ ] Kindly  change the name of your `getBook` function to `fetchBook`. As according to the [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/connect_to_api.md#project-requirements) your function should be called `fetchBook`

> Create a special action creator using [createAsyncThunk](https://redux-toolkit.js.org/api/createAsyncThunk) called `fetchBooks`, which includes the HTTP request with `axios`
> Add `fetchBooks` to your the extraReducers section of your `booksSlice`, where you will actually update the state